docs/index.rst

```diff
@@ -89,5 +89,15 @@ JAX is Autograd_ and XLA_, brought together for high-performance numerical compu
    glossary
 
 
+The core JAX library focuses on scalable array operations.
+If you're looking to train neural networks, use flax_ and start with its documentation.
```

Comment:
Link to its documentation? Maybe https://flax.readthedocs.io/en/latest/

Reply:
The trailing underscore makes this a link (the URL is below)

Reply:
Ah yes, but I assumed the link was to the flax github repo, and so I was suggesting that we link its documentation under the "its documentation" text.

How about we delete the "and start with its documentation" part then?